radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -285,21 +287,25 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 }
 
 /**
+
  * Resets microcontroller of the antenna
  * Cancels the current deployement of the Antenna systems
  * @return 0 for success, non-zero for failure. See hal/errors.h for details.
  */
```

Comment:
What I meant by add white space is that your func doc isn't formatted like the ones found on the [readme](https://github.com/USST-RADSAT-SK/software-and-command#functions). we have two types of func docs [pick one here](https://github.com/USST-RADSAT-SK/software-and-command#functions).